SharedPackages/BrowserServicesKit/Sources/Bookmarks/FaviconsFetcher/BookmarksFaviconsFetcher.swift

```diff
@@ -236,7 +237,7 @@ public final class BookmarksFaviconsFetcher {
     private let database: CoreDataDatabase
     private let stateStore: BookmarksFaviconsFetcherStateStoring
     private let fetcher: FaviconFetching
-    private let faviconStore: FaviconStoring
+    private let faviconStore: @MainActor () async -> FaviconStoring
```

Comment:
This now becomes an asynchronous Main Actor closure to support macOS' FaviconsManager.